pkg/registry.go

```diff
@@ -158,51 +158,80 @@ func (r *ProtocolRegistryImpl) setupProtocolOperations() error {
 		return errors.New("please provide ETH chain config")
 	}
 
+	bscConfig, ok := r.chainConfigs["56"]
```

Comment:
make a variable e.g.; `const eth_main = "1"` and use it like r.chainConfigs[eth_main]